css/login.css

```diff
@@ -36,25 +48,39 @@ body {
   padding: var(--size-3) var(--size-6);
 }
 
-.login a {
+.login .forgot_pwd {
   margin-top: var(--size-2);
   align-self: center;
   transition: 0.5s;
   color: var(--primary-color);
 }
 
-.login a:hover {
+.login .forgot_pwd:hover {
   text-decoration: underline;
   cursor: pointer;
 }
 
-hr {
+.login_pwd {
+  position: relative;
+}
+
+.login_pwd .visibility {
+  position: absolute;
+  right: 3%;
+  top: 45%;
+}
+
+.hr {
   border: none;
   border-top: 2px solid var(--gray-700);
   margin-top: var(--size-3);
   margin-bottom: var(--size-3);
 }
 
+.btn_icon:hover {
+  cursor: pointer;
+}
+
 .btn_create__acc {
   background-color: #f0a428;
 }
```

Comment:
use variables for color codes

Reply:
Yes, this is just for time being, will do it afterwards.